.travis.yml

```diff
@@ -34,12 +32,14 @@ addons:
     - pkg-config
     - pmccabe
     - gawk
+before_deploy:
+    - mv out/Tetris out/Tetris_$TRAVIS_OS_NAME_$CXX
```

Comment:
How would you parse this line if you were the interpreter? Does this work? I think braces should be used to enclose variable name like in `sh`.

`mv out/Tetris out/Tetris_${TRAVIS_OS_NAME}_${CXX}`

Indent seems to be wrong anyways.